### PR TITLE
Updated storage pool section

### PR DESCRIPTION
**Pull request by ranjithwingrider**
Added the storage pool creation using cstor and modified the existing heading for Jiva.
This will update the page for pool creation with cStor.

Signed-off-by: ranjithwingrider ranjith.raveendran@openebe.io

### CODE REVIEW

File: docs/setupstoragepools.md [hunk 1]
```diff
@@ -27,8 +29,9 @@ Attach the disk to the node. Use the following command to attach the disk to a p
 gcloud compute instances attach-disk <Node Name> --disk disk1 --zone us-central1-a
 ```
 
-## Configuring a Storage Pool on OpenEBS
+## Configuring a Storage Pool on OpenEBS using Jiva
 
+Using Jiva, you can create storage pool on hosted path or an external disk. You can add single external disk such as GPD,local disk,etc to your Nodes as mentioned in the above [section](/docs/next/setupstoragepools.html#creating-and-attaching-a-disk-on-gke-node).
 To utilize an external disk, you must create a storage pool on the node where you have attached the disk. Use the following command to login to the node from master. Replace `< Node Name>` with your actual node nmae and `< Zone Of Your Node >` with the actual zone.
```
Comment (mahebbar) [inline docs/setupstoragepools.md]: `<Node Name>`

File: docs/setupstoragepools.md [hunk 2]
```diff
@@ -79,6 +82,87 @@ spec:
...
+    cas.openebs.io/config: |
+      - name: StoragePoolClaim
+        value: "cstor-disk"
+provisioner: openebs.io/provisioner-iscsi
+---
+```
+
 ## Scheduling a Pool on a Node
 
 If you want to schedule your pool on a particular node please follow [this](https://docs.openebs.io/docs/next/scheduler.html) procedure before applying the *openebs-operator.yaml* file. Please refer [installation](/docs/next/installation.html#install-openebs-using-kubectl) to deploy OpenEBS cluster in your k8s environment which will create a default storage pool on the host path.Jiva volumes will be deployed inside this path only.
```
Comment (mahebbar) [inline docs/setupstoragepools.md]: path. Jiva volumes...

File: docs/setupstoragepools.md (hunk 1 above)
Comment (mahebbar) [reply docs/setupstoragepools.md]: node nmae = node name

File: docs/setupstoragepools.md (hunk 2 above)
Comment (ranjithwingrider) [reply docs/setupstoragepools.md]: Done

File: docs/setupstoragepools.md (hunk 1 above)
Comment (ranjithwingrider) [reply docs/setupstoragepools.md]: Done

File: docs/setupstoragepools.md [hunk 3]
```diff
@@ -27,8 +29,9 @@ Attach the disk to the node. Use the following command to attach the disk to a p
 gcloud compute instances attach-disk <Node Name> --disk disk1 --zone us-central1-a
 ```
 
-## Configuring a Storage Pool on OpenEBS
+## Configuring a Storage Pool on OpenEBS using Jiva
 
+Using Jiva, you can create storage pool on hosted path or an external disk. You can add single external disk such as GPD,local disk,etc to your Nodes as mentioned in the above [section](/docs/next/setupstoragepools.html#creating-and-attaching-a-disk-on-gke-node).
```
[the remark this answers is not in the document]
Comment (ranjithwingrider) [reply docs/setupstoragepools.md]: Done

**Pull request merged**